ddtrace/tracer/remote_config.go

```diff
@@ -158,18 +159,33 @@ func (t *tracer) onRemoteConfigUpdate(u remoteconfig.ProductUpdate) map[string]s
 	return statuses
 }
 
-// startRemoteConfig starts the remote config client
-// and registers the APM_TRACING product and its callback.
+// startRemoteConfig starts the remote config client.
+// It registers the APM_TRACING product with a callback,
+// and the LIVE_DEBUGGING product without a callback.
 func (t *tracer) startRemoteConfig(rcConfig remoteconfig.ClientConfig) error {
 	err := remoteconfig.Start(rcConfig)
 	if err != nil {
 		return err
 	}
-	return remoteconfig.Subscribe(
+
+	var dynamicInstrumentationError, apmTracingError error
+
+	if t.config.dynamicInstrumentationEnabled {
+		dynamicInstrumentationError = remoteconfig.RegisterProduct(state.ProductLiveDebugging)
+	}
+
+	apmTracingError = remoteconfig.Subscribe(
 		state.ProductAPMTracing,
 		t.onRemoteConfigUpdate,
 		remoteconfig.APMTracingSampleRate,
 		remoteconfig.APMTracingHTTPHeaderTags,
 		remoteconfig.APMTracingCustomTags,
 	)
+
+	if apmTracingError != nil || dynamicInstrumentationError != nil {
+		return fmt.Errorf("could not subscribe to at least one remote config product: %w, %w",
+			apmTracingError, dynamicInstrumentationError)
+	}
```

Comment:
To make sure we're keeping the same behaviour from before, we should return `apmTracingError` and only log `dynamicInstrumentationError`.

I'm happy to change this if we understand what it means to return error from this function 🙂 (or when / if registering a product can fail).

Reply:
Sorry, missed your other comment.

If the error is just logged then this is fine :+1: